### PR TITLE
Fix various typos in the documentation

### PR DESCRIPTION
**Pull request by aoot**
Various on going fix suggestions.

### CODE REVIEW

**Comment by heplesser** [comment]
@aoot The `vale` test currently fails due to an incorrect build configuration. This is fixed in master. Could you pull master into your branch?

**Review by heplesser — APPROVED** [review]
Formalities are ok, approving.

File: doc/htmldoc/synapses/connection_management.rst [hunk 1]
```diff
@@ -588,20 +588,20 @@ columns and :math:`m` rows.
 
     # To only connect pairs with a nonzero weight, we use array indexing
     # to extract the weights and post-synaptic neurons.
-          nonzero_indices = numpy.where(weights != 0)[0]
-          weights = weights[nonzero_indices]
-          post = B[nonzero_indices]
+        nonzero_indices = np.where(weights != 0)[0]
```
Comment (jessica-mitchell) [inline doc/htmldoc/synapses/connection_management.rst:591]: I'd argue that these should remain ad numpy, for the reason that it's explicit. We don't set up `import numpy as np` anywhere in the code examples on this page. So I'd suggest changing the first `np.array` to `numpy.array`

**Review by jessica-mitchell — CHANGES_REQUESTED** [review]
Thanks @aoot for contributing to our documentation. It's very appreciated. I just have one comment that needs addressing.

File: doc/htmldoc/neurons/parametrization.rst [hunk 2]
```diff
@@ -470,7 +470,7 @@ Using parameters makes it easy to set node properties
 |                                               |                                                    |
 |     for gid in nrns:                          |     nrns.V_m=nest.random.uniform(-20., 20)         |
```
Comment (nicolossus) [inline doc/htmldoc/neurons/parametrization.rst:471]:
```suggestion
|     for gid in nrns:                          |     nrns.V_m = nest.random.uniform(-20.0, 20.0)    |
```
Might as well fix the formatting here also.

File: doc/htmldoc/neurons/parametrization.rst [hunk 3]
```diff
@@ -470,7 +470,7 @@ Using parameters makes it easy to set node properties
...
 |       v_m = numpy.random.uniform(-20., 20.)   |                                                    |
```
Comment (nicolossus) [inline doc/htmldoc/neurons/parametrization.rst:472]:
```suggestion
|       v_m = numpy.random.uniform(-20.0, 20.0) |                                                    |
```
Black compliant formatting.

File: doc/htmldoc/neurons/parametrization.rst [hunk 4]
```diff
@@ -470,7 +470,7 @@ Using parameters makes it easy to set node properties
...
-|       nest.SetStatus([node_id], {'V_m': V_m}) |                                                    |
+|       nest.SetStatus(gid, {'V_m': v_m})       |                                                    |
```
Comment (nicolossus) [inline doc/htmldoc/neurons/parametrization.rst:473]:
```suggestion
|       nest.SetStatus(gid, {"V_m": v_m})       |                                                    |
```

File: models/glif_cond.h [hunk 5]
```diff
@@ -103,8 +103,8 @@ parameter setting of voltage_reset_fraction and voltage_reset_add may lead to th
 situation that voltage is bigger than threshold after reset. In this case, the neuron
 will continue to spike until the end of the simulation regardless the stimulated inputs.
 We recommend the setting of the parameters of these three models to follow the
-condition of :math:`(E_L + voltage_reset_fraction * ( V_th - E_L ) + voltage_reset_add)
-< (V_th + th_spike_add)`.
+condition of :math:`(E_L + \text{voltage_reset_fraction} * ( V_{th} - E_L ) + \text{voltage_reset_add})
+< (V_{th} + \text{th_spike_add})`.
```
Comment (nicolossus) [inline models/glif_cond.h:107]:
```suggestion
condition of 

.. math::

    E_L + \mathrm{voltage_reset_fraction} \cdot \left( V_\mathrm{th} - E_L \right)
    + \mathrm{voltage_reset_add} < V_\mathrm{th} + \mathrm{th_spike_add}
```
I think this should be in a proper math block.

**Review by nicolossus — CHANGES_REQUESTED** [review]
@aoot Thanks for the contribution. I have some small comments. I also agree with @jessica-mitchell that `numpy.foo` instead of `np.foo` should be used to be explicit.

File: doc/htmldoc/synapses/connection_management.rst (hunk 1 above)
Comment (aoot) [reply doc/htmldoc/synapses/connection_management.rst]: I agree with your thought process, thanks for the callout.
There was another line prior to this line that was using `np` and I've amended that to `numpy` as well.

**Review by jessica-mitchell — APPROVED** [review]

**Comment by jessica-mitchell** [comment]
@aoot could you please resolve the conflicts and commit the suggestion from @nicolossus? Once that's sorted we can probably merge this PR, thanks :)

**Comment by aoot** [comment]
Thank you for your patience! I've resolved the conflicts.

**Pull request merged**